apps/api/src/modules/user/user.controller.ts

```diff
@@ -26,4 +28,24 @@ export class UserController {
   getUserData(@User() user: JwtDto) {
     return this.userService.getUserData(user.userId)
   }
+
+  @Get('pending')
+  @UseTypeAuthGuard(UserType.ADMIN)
+  @ApiOperation({ summary: 'get all user pending for admin' })
+  @ApiOkResponse({ type: GetUserDto, isArray: true })
+  getPendingUser() {
+    return this.userService.getPendingUsers()
+  }
+
+  @Put(':id/verification')
+  @UseTypeAuthGuard(UserType.ADMIN)
+  @ApiOperation({ summary: 'accept or reject user with id' })
+  @ApiOkResponse({ type: GetUserDto })
+  @ApiNotFoundResponse({ description: 'user not found' })
+  updateUserStatus(
+    @Param('id') id: number,
```

Comment:
Param return string instead of number ?
```suggestion
    @Param('id') id: string,
```